svm/src/transaction_processor.rs

```diff
@@ -868,7 +877,7 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
                 log_messages,
                 inner_instructions,
                 fee_details: loaded_transaction.fee_details,
-                is_nonce: loaded_transaction.nonce.is_some(),
+                is_nonce: loaded_transaction.rollback_accounts.nonce().is_some(),
```

Comment:
nit: actually, `TransactionExecutionDetails::is_nonce` can be removed even before this pr?

Reply:
Not before this PR because it's required in `filter_program_errors_and_collect_fee`.. but it can be removed in this PR actually